controllers/actions.github.com/resourcebuilder.go

```diff
@@ -84,6 +48,62 @@ func SetListenerImagePullPolicy(pullPolicy string) bool {
 
 type resourceBuilder struct{}
 
+func (b *resourceBuilder) newAutoScalingListener(autoscalingRunnerSet *v1alpha1.AutoscalingRunnerSet, ephemeralRunnerSet *v1alpha1.EphemeralRunnerSet, namespace, image string, imagePullSecrets []corev1.LocalObjectReference) (*v1alpha1.AutoscalingListener, error) {
```

Comment:
Ordered methods so that `newX` is close to its related methods.
It should follow the order of creation: listener, listener resources, ephemeral runner set, ephemeral runner, ephemeral runner resources...

Reply:
I assume there is no real code change other than moving methods around? (maybe changes to use new const)